src/node.cpp

```diff
@@ -1837,12 +1818,10 @@ bool LocalNode::serialize(string* d)
         d->append((const char*)buf, Serialize64::serialize(buf, mtime));
     }
 
-    const int8_t syncableInt = syncable ? 1 : 0;
-    const int8_t syncableIntByteCount = sizeof(syncableInt);
-    d->append((char*)&syncableIntByteCount, 1);
-    d->append((char*)&syncableInt, syncableIntByteCount);
+    const char syncable = mSyncable ? 1 : 0;
+    d->append(&syncable, sizeof(syncable));
 
-    d->append("\0\0\0\0\0\0", 7); // Use these bytes for extensions
+    d->append("\0\0\0\0\0\0", 8); // Use these bytes for extensions
```

Comment:
We think that better to add 8 bytes for extensions, rather than 7, now that we include them for localnodes too :)